src/thirdparty/bcd/bcd/Utils.h

```diff
@@ -26,6 +30,15 @@
 namespace bcd
 {
 
+inline bool isFinite(const float x)
+{
+#ifdef _WIN32
+    return _finite(x) != 0;
+#else
+    return !(std::isnan(x) || std::isinf(x));
```

Comment:
I'm afraid that won't compile with VS 2012.

Reply:
On VS 2012 we use _finite(). We tested with sergo before that VS2012 supported it.